scripts/test_examples.sh

```diff
@@ -201,44 +201,457 @@ stopKafka
 
 startCustomizedKafka examples/kafka-plugin/manual-partitioner/server.properties
 
-expected=("Sending 10 sync Kafka notifications
-Sending 10 async Kafka notifications
-Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '5', key: 'proto-key',
-Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '6', key: 'proto-key',
-Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '7', key: 'proto-key',
-Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '8', key: 'proto-key',
-Received sync Kafka Message, topic 'example-sync-topic', partition '1', offset '9', key: 'proto-key',
-Received async Kafka Message, topic 'example-async-topic', partition '2', offset '5', key: 'async-proto-key',
-Received async Kafka Message, topic 'example-async-topic', partition '2', offset '6', key: 'async-proto-key',
-Received async Kafka Message, topic 'example-async-topic', partition '2', offset '7', key: 'async-proto-key',
-Received async Kafka Message, topic 'example-async-topic', partition '2', offset '8', key: 'async-proto-key',
-Received async Kafka Message, topic 'example-async-topic', partition '2', offset '9', key: 'async-proto-key',
-Async message successfully delivered, topic 'example-async-topic'
-Sync watcher closed
-Async watcher closed
+# Let us test parameters display
+expected=("Usage of examples/kafka-plugin/manual-partitioner/manual-partitioner:
+  -config-dir=\".\": Location of the configuration files; also set via 'CONFIG_DIR' env variable.
+  -kafka-config=\"kafka.conf\": Location of the kafka Client configuration file; also set via 'KAFKA_CONFIG' env variable.
+  -logs-config=\"logs.conf\": Location of the configuration files; also set via 'LOGS_CONFIG' env variable.
+  -messageCount=\"\": Number of messages which will be send. Set to '0' to just watch
+  -microservice-label=\"vpp1\": microservice label; also set via 'MICROSERVICE_LABEL' env variable.
+  -offsetMsg=\"\": Use 'latest', 'oldest' or exact number of message offset
+")
+
+cmd="examples/kafka-plugin/manual-partitioner/manual-partitioner --kafka-config examples/kafka-plugin/manual-partitioner/kafka.conf -h"
+testOutput "${cmd}" "${expected}"
+
+# Let us test the running without parameters - in example are generated 10 Kafka Messages to both topics but the consumed is only 5 messages from each topic beginning with offset 5
+expected=("Invalid partition -1 - defaulting to 0
```

Comment:
please do not expect log messages which are not called in the example main.go. Check everywhere